deluge/ui/baseargparser.py

```diff
@@ -117,9 +182,24 @@ def __init__(self, *args, **kwargs):
                                 help=_("Profile %(prog)s with cProfile. Outputs to stdout "
                                        "unless a filename is specified"))
 
-    def parse_args(self, *args):
-        options, remaining = super(BaseArgParser, self).parse_known_args(*args)
+    def parse_args(self, args=None):
+        """Parse UI arguments and handle common and process group options.
+
+        Unknown arguments return an error and resulting usage text.
+        """
+        options = super(BaseArgParser, self).parse_args(args=args)
+        return self._parse_args(options)
+
+    def parse_known_ui_args(self, args=None):
+        """Parse UI arguments and handle common and process group options.
```

Comment:
Perhaps `Parse UI arguments and handle common and process group options without error.` will do